core/actionProxy/actionproxy.py

```diff
@@ -139,9 +142,9 @@ def error(msg):
 
         # stdout/stderr may be either text or bytes, depending on Python
         # version.   In the latter case, decode to text.
-        if isinstance(o, bytes):
+        if not isinstance(o, str):
```

Comment:
@jtyberg the changes made pr #1867 caused a regression in swift actions (we had a test disabled and didn't catch the regression). This pr fixes the regression and enables the test. But I'm not sure that we have this right for the two different python versions we're supporting. Should we explicitly just check the sys.version_info.major and predicate the decoding?

FYI @cclauss